activities/tool-management/jitsi-guides.md

```diff
@@ -0,0 +1,45 @@
+---
+type: guide
+explains: account creation on our community instance of Jitsi Meet
+---
+
+# Jitsi Meet guides
+
+Jitsi Meet is our videocall and conference tool. Next to our internal Jitsi Meet, we also have a Jitsi Meet instance running for our community on 'meet.community.publiccode.net'. Users can [sign up for an account](https://meet.community.publiccode.net/accountmanager/register/) so they can host meetings themselves.
+
+## Jitsi Meet user guide
+
+* A user signs up by filling out the [registration form](https://meet.community.publiccode.net/accountmanager/register/)
```

Comment:
Since these are steps to be taken in a specific order, a numbered list should be used instead of bullets (which we use for unordered lists).

Reply:
Unlike bullets, items in a numbered lists do have terminal punctuation.